lms/djangoapps/certificates/models.py

```diff
@@ -568,13 +569,18 @@ def certificate_info_for_user(user, course_id, grade, user_is_whitelisted, user_
     """
     Returns the certificate info for a user for grade report.
     """
+    from student.models import CourseEnrollment
+
     certificate_is_delivered = 'N'
     certificate_type = 'N/A'
     status = certificate_status(user_certificate)
     certificate_generated = status['status'] == CertificateStatuses.downloadable
     can_have_certificate = CourseOverview.get_from_id(course_id).may_certify()
+    enrollment_mode, __ = CourseEnrollment.enrollment_mode_for_user(user, course_id)
+    mode_is_audit = enrollment_mode in CourseMode.AUDIT_MODES
+    user_is_verified = grade is not None and not mode_is_audit
```

Comment:
Rather than checking for `not is audit` what are your thoughts on checking `CourseMode.is_verified_mode` and get rid of the negation? 
I think it makes is more readable. 
```
mode_is_verified = CourseMode.is_verified_mode(enrollment_mode)
user_is_verified = grade is not None and mode_is_verified
```